src/core/XRayHeatingFunctions.c

```diff
@@ -1428,7 +1440,7 @@ double dT_comp(double z, double TK, double xe)
   double Trad, ans;
 
   Trad = TCMB * (1.0 + z);
-  ans = (-1.51e-4) * (xe / (1.0 + xe)) / (hubble((float)z) / (HUBBLE * run_globals.params.Hubble_h)) /
+  ans = (-1.51e-4) * (xe / (1.0 + xe + f_He)) / (hubble((float)z) / (HUBBLE * run_globals.params.Hubble_h)) /
```

Comment:
Brad was doing some debugging on the Ts calculation and noticed some difference between 21cmFAST and Meraxes. Despite all other differences between the two, this is another one that we didn't notice...